src/gd.h

```diff
@@ -76,7 +76,16 @@ extern "C" {
 # define BGD_STDCALL
 #endif
 
+#if defined(__GNUC__) || defined(__clang__)
+# define BGD_ATTR_DEPRECATED __attribute__((__deprecated__))
+#elif defined(_MSC_VER)
+# define BGD_ATTR_DEPRECATED __declspec(deprecated)
+#else
+# define BGD_ATTR_DEPRECATED
+#endif
+
 #define BGD_DECLARE(rt) BGD_EXPORT_DATA_PROT rt BGD_STDCALL
+#define BGD_DEPRECATED(rt) BGD_ATTR_DEPRECATED BGD_EXPORT_DATA_PROT rt BGD_STDCALL
```

Comment:
can't you write:
`#define BGD_DEPRECATED(rt) BGD_ATTR_DEPRECATED BGD_DECLARE(rt)`

Reply:
Good catch! Will fix that later.